packages/prisma-client/prisma/migrations/20240127230238_project-preview/migration.sql

```diff
@@ -0,0 +1,26 @@
+-- AlterTable
+ALTER TABLE
+  "Project"
+ALTER COLUMN
+  "previewImageAssetId"
+SET
+  NOT NULL;
+
+-- Drop DashboardProject View
+DROP VIEW IF EXISTS "DashboardProject";
+
+-- Update DashboardProject View
+CREATE VIEW "DashboardProject" AS
```

Comment:
use ‘create or replace view’ instead of drop/create

Reply:
Left join assetId with assets table to not use all that logic with assets.findMany

Reply:
or see other comment, assetId is relation like user_id user etc